dashboard/config/levels/custom/turtle/frozen circle function with parameter.level

```diff
@@ -1,7 +1,8 @@
 <Artist>
   <config><![CDATA[{
+  "published": true,
   "game_id": 23,
-  "created_at": "2014-10-28T00:15:42.000Z",
+  "created_at": "2022-02-16T19:37:58.000Z",
```

Comment:
I don't know if it really matters but do we want to change the created_at dates?

Reply:
Oh yeah, that's weird. FWIW, I've updated other levels in this manner before and I've not been aware of any problems. Happy to manually revert those lines though if you think it's worthwhile! I was almost going to cut all the changes to audit log and other properties too, just to make the diff smaller.

Reply:
Yeah I don't think it causes a problem, it's just metadata that we might want to keep. This happens because on your local instance you "created" the levels in 2022 (it's whenever you first seeded I think)

Reply:
That makes sense!

Reply:
I manually revert all changes except to callouts_json.